util/src/main/java/tc/oc/pgm/util/xml/XMLUtils.java

```diff
@@ -395,15 +398,23 @@ public static <T extends Number & Comparable<T>> T parseNumberInRange(
    */
   public static <T extends Number & Comparable<T>> Range<T> parseNumericRange(
       Node node, Class<T> type) throws InvalidXMLException {
-    Matcher matcher = RANGE_RE.matcher(node.getValue());
+    String nodeValue = node.getValue();
+    Matcher matcher = RANGE_RE.matcher(nodeValue);
     if (!matcher.matches()) {
-      T value = parseNumber(node, node.getValue(), type, true);
+      Matcher legacy = RANGE_LEGACY.matcher(nodeValue);
+      if (legacy.matches()) {
+        String[] split = nodeValue.split("\\.\\.");
+        T lower = parseNumber(node, split[0], type, false);
+        T higher = parseNumber(node, split[1], type, false);
+        return Range.closed(lower, higher);
+      }
```

Comment:
I'd be nice if this supported `5..` for "5 or more" and `..5` for "5 or less", currently this requires both sides. Also, because you're using `\\d+`, it does not support `oo` for infinity which is generally accepted by pgm, so i suggest you change to `\\s` like the other range pattern does

Reply:
was the "5 or more" thing supported before? As I've understood it its a legacy format.
Ill do the infinity thing

Reply:
It probably wasn't, but personally i find the `5..` or `..5` syntax more intuitive than `(oo, 5]`